llvm/test/CodeGen/RISCV/stack-guard-global.ll

```diff
@@ -0,0 +1,47 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=riscv64-unknown-elf -verify-machineinstrs < %s \
```

Comment:
This was previously linux-gnu not elf. If it works fine with a generic triple like that then it is preferred, but we also prefer shortening such triples to just riscv64, as the rest is implied.